src/styles/components/_home.scss

```diff
@@ -17,7 +17,7 @@ $breakpoint-top-section-scale-up: 70rem;
     padding: 0 1.25rem; // top&bottom: 0; left&right: 20px/16
     background-color: var(--background-color);
     color: var(--foreground-color);
-    max-width: 100rem; // 1600px/16
+    max-width: 79.625; // 1274px/16
```

Comment:
I agree, things have gotten a bit off kilter on this page in terms of the other sections lining up. It's interesting because when you look at the two side by side, this top section does look a lot like the design. If you hone in on the issue, it's that the grid margin is not as large as it should be w/ reference to the Figma design. 

Here is our version:
![image](https://user-images.githubusercontent.com/49820744/203467374-63d20ede-bb26-44e7-8a8d-70711df772d1.png)

Here is the figma design:
![image](https://user-images.githubusercontent.com/49820744/203467412-3547ceb6-36db-4075-ad8a-1f15f7e1feeb.png)

The size of the image is also huge compared to the design. This is out of scope here, but I agree we should address that (and the issues w/ breakpoints at larger sizes) in another card. 

But breakpoints specified in this card look like they are now working as they should if you take into acct. that the left margin is too small in our design.  Thanks for pointing out the missing "rem" on line 20...I did fix that! :)

Reply:
I also think maybe me being able to rebase this will help. Since I'm tweaking a breakpoint I want to make sure what I did still works w/ the changes you pushed up! I am going to give it another try tomorrow and reach out to Dustin for help if I get stuck. :)

Reply:
Sorry, I might not be explaining myself well. I think what you're referring to is part of my `style-fixes` branch. The header is changing at one screen size, then the grid is changing at another, so I'm taking care of that. 

In this case, specifically with the newly added `max-width: 79.625; // 1274px/16` instead of `max-width: 100rem; // 1600px/16`, we get something like this, where this hero you're working on just doesn't match the other `max-widths` on the rest of the page: 
<img width="1197" alt="Screen Shot 2022-11-23 at 9 14 52 AM" src="https://user-images.githubusercontent.com/69602589/203569000-bdc7ca25-935e-4a13-a05c-59438b1b8f79.png">

Reply:
Let's hop on a zoom today to clarify so I'm sure what you meant. I'll send you a slack msg--thanks! :)

Reply:
I think it will help when I have your changes. The figma above shows a larger left gap and I matched it to that--hoping to get in that rebase soon and then I think this will all make more visual sense! :)

Reply:
Hey, the view is the same on the main preview but if you compare w/ the view now you can see where I fixed the stacking in that screen size range (per this card)

![image](https://user-images.githubusercontent.com/49820744/203587296-9c751a34-dd28-41c1-8f1a-668e2880c5c3.png)